project/Newevaluation.html

```diff
@@ -6,85 +6,118 @@
   <style>
     body{
       background-color: rgb(82, 86, 89);
-      margin-top: 50px;
+      margin-top: 3%;
```

Comment:
credeam ca trebuie sa folosim unitatile de masura specifice design-ului responsive si la margin si padding. Adica ma gandeam asa (inainte sa aflu ieri de media query): daca pun pixeli, atunci marginea respectiva trebuie sa fie respectata si pe ecran mic, prin urmare s-ar strica lucrurile. At least now I know. I'll change my margins and paddings :D